tools/c7n_gcp/c7n_gcp/resources/compute.py

```diff
@@ -617,3 +617,23 @@ class resource_type(TypeInfo):
         def get(client, resource_info):
             return client.execute_command(
                 'get', {'project': resource_info['project_id']})
+
+
+@resources.register('dataproc-clusters')
+class DataprocClusters(ChildResourceManager):
+
+    class resource_type(ChildTypeInfo):
+        service = 'dataproc'
+        version = 'v1'
+        component = 'projects.regions.clusters'
+        enum_spec = ('list', 'clusters[]', None)
+        scope_key = 'projectId'
+        name = id = 'name'
+        parent_spec = {
+            'resource': 'region',
+            'child_enum_params': {
+                ('name', 'region')},
+            'use_child_query': True,
+        }
+        default_report_fields = ['id', 'name', 'dnsName', 'creationTime', 'visibility']
```

Comment:
we also have two additional required fields on metadata, one for asset inventory type, and one around urn construction. the later is a c7n specific construct that resembles an arn. I added support for both into the data fusion pr by way of example

Reply:
done. Please check